src/words.ts

```diff
@@ -0,0 +1,163 @@
+import {mmod} from 'xen-dev-utils';
+
+type StepVector = Record<string, number>;
```

Comment:
We had some git issues, so we're doing a co-commit. Just minor polish remains like exporting this type so that it's visible to TypeDoc and package users.